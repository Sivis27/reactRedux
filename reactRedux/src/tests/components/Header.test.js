import React from 'react'
import { shallow } from 'enzyme'

//import Header from '../../components/Header/index'
import Header from '../../components/Header';

// test('Should render Header correctly', () => {
//   const wrapper = shallow(<Header />)
//   expect(wrapper).toMatchSnapshot()
// })

test('should test Header component', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
 });