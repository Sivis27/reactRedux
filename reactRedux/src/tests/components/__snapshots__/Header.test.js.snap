// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Header component 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <a
      className="logo"
      href="/"
    >
      React CRUD App
    </a>
  </div>
</header>
`;
